Makefile

```diff
@@ -86,7 +87,7 @@ srpm:
 	docker run -v `pwd`/dist:/dist:rw rpm_fedora_32 sh -c "cp /root/rpmbuild/SRPMS/* /dist"
 
 $(VENV)/bin/mypy: $(VENV)/
-	$(VENV)/bin/pip install mypy
+	$(VENV)/bin/pip install ".[test]" mypy
```

Comment:
mypy is also a test requirement so you can remove that one then

Reply:
Without ".[test]" a package was not installed, probably because I added the requirement later. Probably removing venv would have solved it also.